src/ebuckets.c

```diff
@@ -1780,6 +1780,59 @@ void ebValidate(ebuckets eb, EbucketsType *type) {
         ebValidateRax(ebGetRaxPtr(eb), type);
 }
 
+eItem ebDefragItem(ebuckets *eb, EbucketsType *type, eItem item, ebDefragFunction *fn) {
```

Comment:
Add assert and verify all the calls to this function that the ExpireMeta of the item is not trash.